backend/src/controllers/CalendarController.js

```diff
@@ -0,0 +1,85 @@
+const models = require("../models");
+
+class CalendarController {
+  static browse = (req, res) => {
+    models.calendar
+      .findAll()
+      .then(([rows]) => {
+        res.send(rows);
+      })
+      .catch((err) => {
+        console.error(err);
+        res.sendStatus(500);
+      });
+  };
+
+  static read = (req, res) => {
+    models.calendar
+      .find(req.params.id)
+      .then(([rows]) => {
+        if (rows[0] == null) {
+          res.sendStatus(404);
+        } else {
+          res.send(rows[0]);
+        }
+      })
+      .catch((err) => {
+        console.error(err);
+        res.sendStatus(500);
+      });
+  };
+
+  static edit = (req, res) => {
+    const calendar = req.body;
+
+    // TODO validations (length, format...)
+
+    calendar.id = parseInt(req.params.id, 10);
+
+    models.calendar
+      .update(calendar)
+      .then(([result]) => {
+        if (result.affectedRows === 0) {
+          res.sendStatus(404);
+        } else {
+          res.sendStatus(204);
+        }
+      })
+      .catch((err) => {
+        console.error(err);
+        res.sendStatus(500);
+      });
+  };
+
+  static add = (req, res) => {
+    const calendar = req.body;
+
+    // TODO validations (length, format...)
+    // artist.id = parseInt(req.params.id, 10);
+
+    // models.artist.find(id);
```

Comment:
Dead code

Reply:
We will delete all the dead code on the final PR